Integrations/integration-Quinc.yml

```diff
@@ -0,0 +1,341 @@
+configuration:
+- display: ""
+  name: ServerName
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: ""
+  name: token
+  defaultvalue: ""
```

Comment:
add `insecure` and `proxy` params.
both must be boolean.
default value for insecure and proxy should be `true`

Reply:
- defaultvalue: 'true'
  display: Trust any certificate (insecure)
  name: insecure
  required: false
  type: 8
- defaultvalue: 'true'
  display: Use system proxy
  name: proxy
  required: false
  type: 8